pkgs/test_core/lib/src/runner/reporter/compact.dart

```diff
@@ -186,10 +186,12 @@ class CompactReporter implements Reporter {
           .listen((_) => _progressLine(_lastProgressMessage ?? '')));
     }
 
-    // If this is the first test to start, print a progress line so the user
-    // knows what's running. It's possible that the active test may not be
-    // [liveTest] because the engine doesn't always surface load tests.
-    if (_engine.active.length == 1 && _engine.active.first == liveTest) {
+    // If this is the first test or suite load to start, print a progress line
+    // so the user knows what's running.
+    if ((_engine.active.length == 1 && _engine.active.first == liveTest) ||
+        (_engine.liveTests.isEmpty &&
+            _engine.activeSuiteLoads.length == 1 &&
+            _engine.activeSuiteLoads.first == liveTest)) {
```

Comment:
I don't think  you want the `liveTests` check here - that would make it not show the loading message for the currently loading test if some test has completed and some are still loading, but none is actively running.

```suggestion
        (_engine.activeSuiteLoads.length == 1 &&
            _engine.activeSuiteLoads.first == liveTest)) {
```

Reply:
Ah, yeah this should have been `_engine.active.isEmpty`, not `_engine.liveTests.isEmpty`.

Reply:
We don't want to remove the check entirely or we'd show info about suites that would have never been put into `_enginge.active` before.